pennylane_lightning_gpu/src/util/CMakeLists.txt

```diff
@@ -1,6 +1,39 @@
-project(lightning_gpu_utils LANGUAGES CXX CUDA)
-set(CMAKE_CXX_STANDARD 20)
+project(lightning_gpu_utils LANGUAGES CUDA CXX)
+enable_language(CUDA)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Why did we need to decrease the C++ standard here?

Reply:
aha, because of cuda only supports up to c++17